BAT.bib

```diff
@@ -26,6 +26,41 @@ @article{amin2008automated
 	publisher={Elsevier}
 }
 
+@misc{Historyo5:online,
```

Comment:
![grafik](https://user-images.githubusercontent.com/7289947/58957712-742b7400-87a1-11e9-9df9-70f9d7f3c42f.png)
 History of computer vision contests won by deep CNNs on GPUs
![grafik](https://user-images.githubusercontent.com/7289947/58957944-07fd4000-87a2-11e9-929d-34f106a38d0d.png)

...
year={2017},
month={03},
organization={{The Swiss AI Lab}},
...